core/src/main/java/com/graphhopper/storage/GraphEdgeIdFinder.java

```diff
@@ -239,7 +239,7 @@ public BlockArea setQueryGraph(QueryGraph queryGraph) {
         }
     }
 
-    private static BBox createBBox(NodeAccess na, EdgeIteratorState edgeState) {
+    public static BBox createBBox(NodeAccess na, EdgeIteratorState edgeState) {
```

Comment:
This would be better to duplicate and keep private

Reply:
Fixed